### PR TITLE
[TASK] v11 upgrade

### PR DESCRIPTION
**Pull request by susannemoog**
- use new icon registry
- return responses in controllers
- use moduleTemplate instead of be module viewhelpers
- use new pagination API
- remove objectManager usages
- use more DI

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,11 +14,13 @@
...
+    "require": {
+        "typo3/cms-core": "11.*@dev",
+        "typo3/cms-beuser": "11.*@dev"
+    },
     "require-dev": {
-        "typo3/cms-core": "10.*.*@dev",
-        "typo3/cms-beuser": "10.*.*@dev",
-        "typo3/cms-install": "10.*.*@dev",
-        "typo3/cms-about": "10.*.*@dev",
+        "typo3/cms-install": "11.*@dev"
```
Comment (fsuter) [inline composer.json:22]: I wanted to use your version, but there's an error here. There's a comma missing at the end of the line.

Comment (susannemoog) [reply composer.json]: Updated and fixed.

Comment (fsuter) [reply composer.json]: Great, thanks!

**Comment by bmoex** [comment]
Thanks for your input! My focus on TYPO3 has been reduced drastically.. Will check this today! I will merge this now and review locally and put a new release tag!

**Pull request merged**